plenum/server/replica.py

```diff
@@ -1804,6 +1813,9 @@ def _request_pre_prepare_if_possible(self, three_pc_key) -> bool:
         # TODO: Using a timer to retry would be a better thing to do
         logger.debug('{} requesting PRE-PREPARE({}) from {}'.
                      format(self, three_pc_key, prepare_senders))
+        # An optimisation can be to request PRE-PREPARE from f+1 or
```

Comment:
Should we have a value in quorums.py for the number of nodes sufficient for each type of MsgReq?